message-queue-lib/src/cmr/message_queue/queue/sqs.clj

```diff
@@ -315,7 +426,9 @@
     [this queue-name msg]
     (let [msg (json/generate-string msg)
           queue-name (normalize-queue-name queue-name)
-          queue-url (get-queue-url sqs-client queue-name)]
+          queue-url (get-queue-url
```

Comment:
Looks like we should back out this change.

Reply:
Done.